src/OpenSage.Game/Scene3D.cs

```diff
@@ -108,7 +108,15 @@ internal Scene3D(Game game, MapFile mapFile, string mapPath, int randomSeed)
             _players = Player.FromMapData(mapFile.SidesList.Players, game.AssetStore).ToList();
 
             // TODO: This is completely wrong.
-            LocalPlayer = _players.FirstOrDefault();
+            var valids = _players.Where(x => x.Side.Contains("Gondor") || x.Side.Contains("USA"));
+            if (valids.Count() > 0)
+            {
+                LocalPlayer = valids.First();
+            }
+            else
+            {
+                LocalPlayer = _players.FirstOrDefault();
+            }
```

Comment:
hack to get a valid faction when jumping to a map in Bfme -> .First() is Civilian and thus no castle will be unpacked

Reply:
fixed